### ISSUE TITLE
Build errors (duplicate symbol) on iOS (xCode 13.2.1)

### ISSUE THREAD

**Issue by dukeluo** [fattmerchantorg/Fattmerchant-iOS-SDK#76]
### Your Environment
Fattmerchant SDK version: 1.4.5
Xcode version:  13.2.1
Platform: iOS
OS version: 11.6 and newer
React Native version: 0.63.2
Pod: 
```sh
  - boost-for-react-native (1.63.0)
  - CocoaAsyncSocket (7.6.5)
  - CocoaLumberjack (3.7.2):
    - CocoaLumberjack/Core (= 3.7.2)
  - CocoaLumberjack/Core (3.7.2)
  - DoubleConversion (1.1.6)
  - Fattmerchant (1.4.5)
```

### Expected Behavior
Success build

### Actual Behavior
![image](https://user-images.githubusercontent.com/26000789/184123609-0160bab8-d4fd-4d29-a0fa-e786e35ac6ac.png)

### Steps to Reproduce
- add `react-native-background-geolocation` to package.json
- add `Fattmerchant` SDK to pod
- build app

### Context
I get a duplicate symbol issue when i introduce `Fattmerchant` SDK to my project. The `libLumberjack.a` [file](https://github.com/fattmerchantorg/Fattmerchant-iOS-SDK/blob/master/fattmerchant-ios-sdk/Cardpresent/ChipDnaMobile/libLumberjack.a) of `Fattmerchant` SDK is conflict with my existed dependency `CocoaLumberjack` for `react-native-background-geolocation` module. 

I think this issue is caused by`Fattmerchant` SDK linked with `CocoaLumberjack` without adding a dependency, could you help to deal with this?

**Comment by thomasvidas** [comment]
This has been fixed in the `2.2.1` release today

**Comment by dukeluo** [comment]
The latest version of the SDK works well. Thanks very much.
